diff --git a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/mytry/L070ButtonsOpMode.java b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/mytry/L070ButtonsOpMode.java
index 4ac1693..c9d9d23 100644
--- a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/mytry/L070ButtonsOpMode.java
+++ b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/mytry/L070ButtonsOpMode.java
@@ -30,6 +30,7 @@ public class L070ButtonsOpMode extends CorbelsTeleOp {
     // When on L070S040, see
     // https://msb-stitt.github.io/ftc31459-26-27-biobuzz/guide/tasks/l070.html#l070s040
     // for what to do here.
+    boolean previousButtonA;
 
     // 'public' to match the parent class. 'public' allows access from any code.
     @Override
@@ -64,6 +65,10 @@ public class L070ButtonsOpMode extends CorbelsTeleOp {
         // When on L070S060, see
         // https://msb-stitt.github.io/ftc31459-26-27-biobuzz/guide/tasks/l070.html#l070s060
         // for what to do here.
+        double leftStickX = gamepad1.left_stick_x;
+        Tracker.publish("stick/leftX", leftStickX);
+        double rightStickX = gamepad1.right_stick_x;
+        Tracker.publish("stick/rightX", rightStickX);
 
         double leftSpeed = -leftStickY;
         Tracker.publish("speed/left", leftSpeed);
@@ -85,10 +90,20 @@ public class L070ButtonsOpMode extends CorbelsTeleOp {
         // When on L070S030, see
         // https://msb-stitt.github.io/ftc31459-26-27-biobuzz/guide/tasks/l070.html#l070s030
         // for what to do here.
+        boolean buttonA = gamepad1.a;
+        Tracker.publish("button/a", buttonA);
 
         // When on L070S050, see
         // https://msb-stitt.github.io/ftc31459-26-27-biobuzz/guide/tasks/l070.html#l070s050
         // for what to do here.
+        if (buttonA != previousButtonA) {
+            if (buttonA) {
+                Tracker.publish("button/event", "A pressed");
+            } else {
+                Tracker.publish("button/event", "A released");
+            }
+        }
+        previousButtonA = buttonA;
 
         loopAfter();
     }
